src/core_plugins/getting_started/public/components/getting_started/getting_started.js

```diff
@@ -0,0 +1,82 @@
+import { uiModules } from 'ui/modules';
+import uiChrome from 'ui/chrome';
+import 'ui/storage';
+import 'ui/filters/trust_as_html';
+import { getTopMessage, getManageAndMonitorMessages } from 'ui/getting_started/registry';
+import { hasOptedOutOfGettingStarted, optOutOfGettingStarted } from 'ui/getting_started/opt_out_service';
+import { documentationLinks } from 'ui/documentation_links';
+import angular from 'angular';
+
+import kibanaLogo from 'ui/images/logo-kibana-small.svg';
+import beatsLogo from 'ui/images/logo-beats-small.svg';
+import logstashLogo from 'ui/images/logo-logstash-small.svg';
+import dashboardIcon from 'ui/images/icon-dashboard.svg';
+import shieldIcon from 'ui/images/icon-shield.svg';
+
+import template from './getting_started.html';
+import './getting_started.less';
+
+function makeAngularParseableExpression(text) {
+  return `<span>${text}</span>`;
+}
+
+const app = uiModules.get('kibana');
+
+app.directive('gettingStarted', function ($injector) {
+
+  const $compile = $injector.get('$compile');
+
+  return {
+    restrict: 'E',
+    template: template,
+    scope: {
+    },
+    bindToController: true,
+    controllerAs: 'gettingStarted',
+    controller: class GettingStartedController {
+      constructor($scope) {
+        if (this.hasOptedOut()) {
+          uiChrome.setVisible(true);
+        } else {
+          uiChrome.setVisible(false);
+        }
+
+        // Compile topMessageHtml and inject it into the DOM
+        const topMessage = getTopMessage();
+        if (topMessage) {
+          const topMessageContainer = document.getElementById('gettingStartedTopMessageContainer');
```

Comment:
I looked at the top nav code and it doesn't seem to use the render directive. It seems to be taking the [`$compile` approach](https://github.com/elastic/kibana/blob/master/src/ui/public/kbn_top_nav/kbn_top_nav_controller.js#L120).

Also, looking into the [definition of the `render` directive](https://github.com/elastic/kibana/blob/master/src/ui/public/render_directive/render_directive.js), it appears to expect a directive definition, not just any arbitrary text (that may or may not contain a directive). This is too constraining for the use case here on the Getting Started page.

Given these considerations, I'm going to stick with the `$compile` approach for the Getting Started page. However, since it wasn't obvious _why_ we were using it here, I'll add some comments explaining the intent. Thanks!